plugins/octant/cmd/antrea-octant-plugin/antrea_info.go

```diff
@@ -39,29 +39,28 @@ const (
 	heartbeatCol      = "Last Heartbeat Time"
 )
 
-func controllerHandler(request service.Request) (component.ContentResponse, error) {
+func (a *antreaOctantPlugin) controllerHandler(request service.Request) (component.ContentResponse, error) {
```

Comment:
p is more proper as the short name as the object is plugin.

Reply:
Changed to "p".